.github/workflows/golangci-lint.yaml

```diff
@@ -11,7 +11,7 @@ permissions:
   contents: read
 
 env:
-  GO_VERSION: v1.23.6
+  GO_VERSION: v1.24.0
```

Comment:
should we jump to 1.24.3 to pick up security fixes: https://go.dev/doc/devel/release#go1.24.3